docs/templates/visualization.md

```diff
@@ -10,9 +10,10 @@ from keras.utils.visualize_util import plot
 plot(model, to_file='model.png')
 ```
 
-`plot` takes one optional arguments:
+`plot` takes two optional arguments:
 
 - `show_shapes` (defaults to False) controls whether output shapes are shown in the graph.
+- `show_layer_names` (deafults to True) controls whether output layers are shwon in the graph.
```

Comment:
That's a lot of typos for one sentence. Also that's not a very accurate description.